test/functional/edition_legacy_associations_controller_test.rb

```diff
@@ -0,0 +1,99 @@
+require 'test_helper'
+
+class Admin::EditionLegacyAssociationsControllerTest < ActionController::TestCase
+  include Admin::EditionRoutesHelper
+
+  should_be_an_admin_controller
+
+  view_test 'should render edit form correctly populated' do
+    @topic = create(:topic)
+    @edition = create(
+      :publication,
+      title: 'the edition',
+      policy_content_ids: ['5d37821b-7631-11e4-a3cb-005056011aef'],
+      topic_ids: [@topic.id.to_s],
+      primary_specialist_sector_tag: 'aaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaa',
+      secondary_specialist_sector_tags: ['aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeee', 'eeeeeeee-bbbb-cccc-dddd-aaaaaaaaaaaaa']
+    )
+    get :edit, params: { edition_id: @edition.id }
+    assert_select "#edition_policy_content_ids option[value='5d37821b-7631-11e4-a3cb-005056011aef'][selected='selected']"
+    assert_select "#edition_topic_ids option[value='#{@topic.id}'][selected='selected']"
+    # assert_select "#edition_primary_specialist_sector_tag option[value='aaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaa'][selected='selected']"
```

Comment:
Is it worth uncommenting this, or perhaps removing it?

Reply:
Actually I forgot to finish the test here, it maybe becomes redundant now as it's covered elsewhere. I'll add it to the card.